pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -516,8 +527,20 @@ func (ctrl *Controller) syncKubeletConfig(key string) error {
 			return ctrl.syncStatusOnly(cfg, err, "could not deserialize the Kubelet source: %v", err)
 		}
 
+		// Get the default API Server Security Profile
+		var profile *configv1.TLSSecurityProfile
+		if apiServerSettings, err := ctrl.apiserverLister.Get(defaultOpenshiftTLSSecurityProfileConfig); err != nil {
+			if !macherrors.IsNotFound(err) {
+				return ctrl.syncStatusOnly(cfg, err, "could not get the TLSSecurityProfile from %v: %v", defaultOpenshiftTLSSecurityProfileConfig, err)
+			}
+		} else {
+			profile = apiServerSettings.Spec.TLSSecurityProfile
+		}
+		if cfg.Spec.TLSSecurityProfile != nil {
+			profile = cfg.Spec.TLSSecurityProfile
+		}
```

Comment:
Is the intent here to always overwrite profile from above when this is set?

Reply:
correct... if the profile is set on the KubeletConfig object then override the global setting.